apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/CoverageTransformerV2.java

```diff
@@ -1098,4 +1099,133 @@ Timer.Context createTimerContext(String partId) {
     return CommonTransformerUtils.createMetricsTimer(
         metricRegistry, getClass().getSimpleName(), "transform", partId);
   }
+
+  /**
+   * concatenates Part C Contract And Pbp.
+   *
+   * @param beneficiary the value for {@link Beneficiary}
+   * @return concatenated Contract And Pbp
+   */
+  private String concatenatePartCContractAndPbp(Beneficiary beneficiary) {
+    int month = Calendar.getInstance().get(Calendar.MONTH) + 1;
+    String contractId = getPartCContractNumber(beneficiary, month).orElse(null);
+    String pbpId = getPartCPbpNumber(beneficiary, month).orElse(null);
+    return contractId + "-" + pbpId;
```

Comment:
Since contractId and pbpId can technically be null, you should have a null check here. Same in the concatenatePartDContractAndPbp() method.

Reply:
@alex-dzeda should we display `Part C` or `Part D` like before in case of null?